test/runtests_parser.py

```diff
@@ -234,12 +234,17 @@ def file_to_obj(filename):
 
     return results
 
-def main():
+def find_runtest_file():
+    rtfile = None
     possible_paths = ["./RunTests", "./test/RunTests"]
```

Comment:
Consider making this not dependent on cwd?

Reply:
done